rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/db/NodeSourceData.java

```diff
@@ -73,7 +80,8 @@ public NodeSourceData() {
     }
 
     public NodeSourceData(String nodeSourceName, String infrastructureType, Object[] infrastructureParameters,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Constructor has 9 parameters, which is greater than 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00107)